.github/workflows/release-openshift.yaml

```diff
@@ -32,6 +32,8 @@ jobs:
           ref: main
           path: ${{ matrix.repository }}
           token: ${{ github.token }}
+      - name: Check binary version
+        run: make check-version VERSION=${{ matrix.version }}
```

Comment:
what value does `matrix.version` have?

Reply:
It is the version of the releasable binary, which is what we want to check matches the binary version output.

Reply:
Actually you are right, that was a copy paste that did not go well.
Fixing

Reply:
fixed, I wonder if there is a safe way to test this one. I managed to test the other 2 releases.

Reply:
I am going to withdraw the changes here, only release changes for the tested workflows which are actually in use now.
This workflow is not currently being used and, when we plam to start using it, it will need further preparations anyway, so no need to prematurely change it now.

Reply:
sgtm 👍